src/rdiff_backup/selection.py

```diff
@@ -402,6 +402,7 @@ def incr_warnings(exc):
         something_excluded, tuple_list = None, []
         separator = Globals.null_separator and b"\0" or b"\n"
         for line in filelist_fp.read().split(separator):
```

Comment:
Quickly, I'm wondering why we are not using `readlines()` or even `for line in f` for that mather.

Reply:
I see it to support the `null_separator`. Was not aware of this feature. Since when can we have carriage return in a filename !? If that tru, we should probably enabled this feature by default.

Any how, This is good for me.

Reply:
The problem is that null characters are not that easy to enter and filenames with carriage return aren't that often.